projects/kit/components/select/native-select/native-select.template.html

```diff
@@ -4,9 +4,10 @@
     [disabledItemHandler]="disabledItemHandler || itemsHandlers.disabledItemHandler"
 ></tui-data-list-wrapper>
 <option
-    *ngFor="let option of items"
-    [value]="option"
+    *ngFor="let option of items; let index = index"
+    [value]="stringify(option)"
+    [selected]="selected(option)"
     [disabled]="disabledItemHandler ? disabledItemHandler(option) : itemsHandlers.disabledItemHandler(option)"
 >
-    {{ option }}
+    {{ stringify(option) }}
```

Comment:
I wonder if we can not write this twice (here and in value binding). But maybe it's good just to be safe in terms of whitespace...